cluster/src/test/java/org/apache/iotdb/cluster/common/TestRemoteFileSnapshot.java

```diff
@@ -56,7 +57,7 @@ public void getRemoteSnapshot() {
     int startTime = 12345;
     for (RemoteTsFileResource tsFileResource : remoteDataFiles) {
       // fake a file for each resource
-      String[] splits = tsFileResource.getFile().getPath().split("\\\\");
+      String[] splits = tsFileResource.getFile().getPath().split(File.separator);
```

Comment:
It is good to take different platforms into account. But `split()` receives a regex so I wonder whether the non-escaped separator string works.
Have you tested it on both Linux and Windows?

Reply:
yes, I have tested it on both Linux and Windows, and it passed.